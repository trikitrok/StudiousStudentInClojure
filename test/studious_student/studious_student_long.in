20
9 rgh woqg dmabatgbt qrvpcrx eluunoi sy w wnthqxgkg aimallazuc
9 jg j uj ujnzdng nzdng nzdn e g ujnzdngj
5 jibw ji jp bw jibw
9 nl jtdmdxu ux nlmnyzdxu mnyz jtdm nlmnyz dxu uxdxu
9 dcn csmzj krnc vkcoume wvpva yqoexwujwp v cxepgptf xb
9 ksdzsjz bbio ja mvvyxzkmq zgdvxolmt xgvwdbfqzn rhubnqtaad qa eeb
9 iccrmcrm mwp sil iccrmcrm ic odo iccrm crm odocrm
9 o zt da wv brorejctww fu phnej ynrdkylwys ekggrmehcl
9 hdfeax d s uxnnrzko nxpcu v njxqbnh aaqzeeb kxpkw
9 vkzzfgtobz i tkczlqaf peqmnyoh eogzpbe bgorlllxor bkkupvnqr gsr xfqmteh
9 wjxwgm qdhmzkmpzv uhibo gcikegpzv ceqiwekdx rxegvkc ujjvbv kfit peiawyk
9 eavcqvv wyuh mkfq not evhlpur eidqnartht pesgphnnlq t ztvu
9 z dvqgfh wqx vnajabkqvs sdwkc dlhcnc ezrcvsc teje gzwwj
9 i hsmh hsmheh xgi eh xg xgeh xnfc ihsmh
9 izqht h qpbdayaifl pjsoie sujccnm umj dralemrspo euswuti m
9 r wwwr ndtc ndtclp lpb b wwwb www lp
9 c evaxdeyrxb rhhfmdm xq vxedern diqs tpdofbc rbq kbxdy
9 k itqsgpwze ma yhpncg xtf w m kahula zgbo
9 fujv mzr kgukjmokvz schpxugnef p rjojpzbsro wpobp wl od
9 u ufmu ufmuqfy vmc ufm uqfy z vmcu qfy