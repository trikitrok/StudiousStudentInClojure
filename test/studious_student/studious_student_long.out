aimallazucdmabatgbteluunoiqrvpcrxrghsywnthqxgkgwoqgw
egjgjnzdngnzdnujnzdngjujnzdnguj
bwjibwjibwjijp
dxujtdmdxujtdmmnyznlmnyzdxunlmnyznluxdxuux
csmzjcxepgptfdcnkrncvkcoumevwvpvaxbyqoexwujwp
bbioeebjaksdzsjzmvvyxzkmqqarhubnqtaadxgvwdbfqznzgdvxolmt
crmiccrmcrmiccrmcrmiccrmicmwpodocrmodosil
brorejctwwdaekggrmehclfuophnejwvynrdkylwyszt
aaqzeebdhdfeaxkxpkwnjxqbnhnxpcusuxnnrzkov
bgorlllxorbkkupvnqreogzpbegsripeqmnyohtkczlqafvkzzfgtobzxfqmteh
ceqiwekdxgcikegpzvkfitpeiawykqdhmzkmpzvrxegvkcuhiboujjvbvwjxwgm
eavcqvveidqnarthtevhlpurmkfqnotpesgphnnlqtwyuhztvu
dlhcncdvqgfhezrcvscgzwwjsdwkctejevnajabkqvswqxz
ehhsmhehhsmhihsmhixgehxgixgxnfc
dralemrspoeuswutihizqhtmpjsoieqpbdayaiflsujccnmumj
blpblpndtclpndtcrwwwbwwwrwww
cdiqsevaxdeyrxbkbxdyrbqrhhfmdmtpdofbcvxedernxq
itqsgpwzekahulakmamwxtfyhpncgzgbo
fujvkgukjmokvzmzrodprjojpzbsroschpxugnefwlwpobp
qfyufmufmuqfyufmuuqfyuvmcuvmcz